### PR TITLE
 add simple reports/post function

### PR DESCRIPTION
**Pull request by hewonjeong**

### CODE REVIEW

**Comment by ochanje210** [comment]
lgtm

**Comment by hewonjeong** [comment]
[[0.5] As a Google Play store administrator, I'd like to assure this app send an abusing report to server.](https://trello.com/c/fV707y4D/174-0-5-as-a-google-play-store-administrator-i-d-like-to-assure-this-app-send-an-abusing-report-to-server)

File: src/server/reports/router.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import express from 'express';
+import Controllers from './controllers';
+
+const router = express.Router();
+router.post('/:id', Controllers.post);
```
Comment (ovekyc) [inline src/server/reports/router.js]: There is no id when post something
Please use just '/'

File: src/test/node/reports/controllers.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import httpMocks from 'node-mocks-http';
+
+test('reports controller post ', t => {
+  const expected = {
+    status: 200,
+    message: 'success'
+  };
+  const req = httpMocks.createRequest({
+    method: 'POST',
+    url: '/reports'
```
Comment (ovekyc) [inline src/test/node/reports/controllers.js]: /api/reports

**Comment by festinalente91** [comment]
LGTM

**Comment by festinalente91** [comment]
adsfdjlasdjflasdlgtmaosdifjoig :)

**Pull request merged**